### PR TITLE
Use absolute path for docs Banner image

### PR DESCRIPTION
**Pull request by matthewhughes934**
Also fix some (seemingly) errant whitespace.

I noticed this while working with the docs recently, and thought I'd have a go at fixing it (i.e. I have no idea if my solution is reasonable)

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -77,11 +78,13 @@ def __getattr__(cls, name):
 
 html_theme_options = {
     "announcement": """
-        <a style=\"text-decoration: none; color: white;\" 
+        <a style=\"text-decoration: none; color: white;\"
            href=\"https://opencollective.com/urllib3\">
-           <img src=\"_static/favicon.png\"/> Sponsor urllib3 v2.0 on Open Collective
+           <img src=\"{docs_root}/{image_path}\"/> Sponsor urllib3 v2.0 on Open Collective
```
Comment (matthewhughes934) [inline docs/conf.py:83]: I realise approach assumes the docs are built and deployed on the same filesystem....which may not necessarily be the case.

**Comment by pquentin** [comment]
@matthewhughes934 Why did you close this?

Also, can you please clarify how you hit this issue in the first place?

**Comment by sethmlarson** [comment]
@pquentin I think this PR is attempting to solve #1984?

**Comment by matthewhughes934** [comment]
@pquentin sorry, yeah I ran into the issue sethmlarson linked. Closed per my comment (with me _assuming_ the docs are built and deployed separately)

**Pull request closed without merging**